datasets/sport-lis.data.mdx

```diff
@@ -0,0 +1,101 @@
+    stacCol: sport-lis-vsm0_100cm-percentile
+    zoomExtent:
+      - 2
+      - 16
+    compare:
+      datasetId: soil-moisture
+      layerId: sport-lis-vsm0_100cm-percentile
+      mapLabel: |
+        ::js ({ dateFns, datetime, compareDatetime }) => {
+          return `${dateFns.format(datetime, 'MMM yyyy')} VS ${dateFns.format(compareDatetime, 'MMM yyyy')}`;
+        }
+---
+
+<Block>
+  <Prose>
+  The NASA Land Information System (LIS) is a high-performance land surface modeling and data assimilation system used to characterize land surface states and fluxes by integrating satellite-derived datasets, ground-based observations, and model re-analyses. The NASA SPoRT Center at MSFC developed a real-time configuration of the LIS (“SPoRT-LIS”), which is designed for use in experimental operations by domestic and international users. SPoRT-LIS is an observations-driven, historical and real-time modeling setup that runs the Noah land surface model over a full CONUS domain. It provides soil moisture estimates at approximately 3-km horizontal grid spacing over a 2-meter-deep soil column and has been validated for regional applications.
+
+  SPoRT-LIS consists of a 33-year soil moisture climatology spanning from 1981 to 2013, which is extended to the present time and forced by atmospheric analyses from the operational North American Land Data Assimilation System-Phase 2. The 33-year soil moisture climatology also provides the database for real-time soil moisture percentiles evaluated for all U.S. counties and at each modeled grid point. The present-day soil moisture analyses are compared to daily historical distributions to determine the soil wet/dry anomalies for the specific time of year. Soil moisture percentile maps are constructed for the model integrated layers, and these data are frequently referenced by scientists and operational agencies contributing to the weekly U.S. Drought Monitor product.  
+
+  The SPoRT-LIS bridges the 4-day latency gap in NLDAS-2 atmospheric forcing input by incorporating forcing from the National Centers for Environmental Prediction Global Data Assimilation System in combination with hourly Multi-Radar Multi-Sensor precipitation estimates. The real-time component of the SPoRT-LIS updates every 6 hours, and also includes a forecast component that produces soil moisture percentiles out to 2 weeks. Another unique feature of SPoRT-LIS is the incorporation of daily, real-time satellite retrievals of Green Vegetation Fraction since 2012, which results in more representative evapotranspiration and ultimately soil moisture estimates than using a fixed seasonal depiction of vegetation in the model. 
+
+  </Prose>
+</Block>
+
+<Block>
+  <Prose>
+    # Interpreting the Data
```

Comment:
Level 1 headings should be avoided in the content as they are reserved for the page title.

```suggestion
    ## Interpreting the Data
```